torch/_dynamo/side_effects.py

```diff
@@ -219,6 +238,10 @@ def check_allowed_side_effect(self, item):
 
     def store_attr(self, item: VariableTracker, name: str, value: VariableTracker):
         assert self.is_attribute_mutation(item)
+
+        if item in self.skip_mutation_variables:
+            return
```

Comment:
Why does this work? If we don't update self.store_attr_mutations, then I expect torch.func.functional_call(mod, new_weight) will not replace mod.weight with new_weight during Dynamo tracing

Reply:
the actual mutations happens seperately with the side effect tracking. E.g. when modifying a dict, the actual mutation happens at https://github.com/pytorch/pytorch/blob/main/torch/_dynamo/variables/dicts.py#L479-L480 after we track the mutation info in sideeffects.

The change here is just that we want to skip the mutation tracking for these variables but not the actual mutation.

Reply:
I think what I mentioned only applies to the dict mutation. The store_attr load_attr is not the case. Let me take a closer look.